lib/services/local/users_test.go

```diff
@@ -69,7 +69,7 @@ func TestRecoveryCodesCRUD(t *testing.T) {
 		require.NoError(t, err)
 
 		// Test fetching of codes.
-		codes, err := identity.GetRecoveryCodes(ctx, username)
+		codes, err := identity.GetRecoveryCodes(ctx, username, true)
```

Comment:
```suggestion
		codes, err := identity.GetRecoveryCodes(ctx, username, true /* withSecrets */)
```

Same throughout, please.